src/utils/fetchTransactions.ts

```diff
@@ -10,7 +10,7 @@ const provider = new ethers.providers.JsonRpcProvider(providerURL)
 
 async function getLatestTxBlockNumber() {
   const txArray = await Transaction.find({}).sort('-blockNumber').limit(1).exec()
```

Comment:
question(non-blocking): Could we use `findOne` with sort here? Might be a little cleaner and quicker to read.